auth0/tenant.yaml

```diff
@@ -60,6 +60,90 @@ resourceServers:
+      - value: patient/Location.read
+        description: Location read access for patient
+      - value: patient/MedicationRequest.read
+        description: MedicationRequest read access for patient
+      - value: patient/Observation.read
+        description: Observation read access for patient
+      - value: patient/Organization.read
+        description: Organization read access for patient
+      - value: patient/Patient.read
+        description: Patient read access for patient
+      - value: patient/Practitioner.read
+        description: Practitioner read access for patient
+      - value: patient/Procedure.read
+        description: Procedure read access for patient
+      - value: patient/Provenance.read
+        description: Provenance read access for patient
+      - value: patient/PractitionerRole.read
+        description: PractitionerRole read access for patient
+        
+      - value: user/*.read
+        description: Read access to the current user
+      - value: user/Medication.read
+        description: Read access to current user's medications
+      - value: user/AllergyIntolerance.read
+        description: Allergy read access for current user
+      - value: user/CarePlan.read
+        description: CarePlan read access for current user
+      - value: user/CareTeam.read
+        description: CareTeam read access for current user
+      - value: user/Condition.read
+        description: Condition read access for current user
+      - value: user/Device.read
+        description: Device read access for current user
+      - value: user/DiagnosticReport.read
+        description: DiagnosticReport read access for current user
+      - value: user/DocumentReference.read
+        description: DocumentReference read access for current user
+      - value: user/Encounter.read
+        description: Encounter read access for current user
+      - value: user/Goal.read
+        description: Goal read access for current user
+      - value: user/Immunization.read
+        description: Immunization read access for current user
+      - value: user/Location.read
+        description: Location read access for current user
+      - value: user/MedicationRequest.read
+        description: MedicationRequest read access for current user
+      - value: user/Observation.read
+        description: Observation read access for current user
+      - value: user/Organization.read
+        description: Organization read access for current user
+      - value: user/Patient.read
+        description: Patient read access for current user
+      - value: user/Practitioner.read
+        description: Practitioner read access for current user
+      - value: user/Procedure.read
+        description: Procedure read access for current user
+      - value: user/Provenance.read
+        description: Provenance read access for current user
+      - value: user/PractitionerRole.read
+        description: PractitionerRole read access for current user  
```

Comment:
Awesome- thank you!  One thing you'll want to look at in your implementation would be to ensure that these scopes are also listed in your smart-configuration metadata file that you serve up to clients.